lisa/sut_orchestrator/azure/features.py

```diff
@@ -1913,10 +1912,13 @@ def enabled(self) -> bool:
 class ArchitectureSettings(schema.FeatureSettings):
     type: str = "Architecture"
     # Architecture in hyper-v
-    arch: str = field(
-        default="x64",
+    arch: str = field(  # type:ignore
```

Comment:
If so, the type should be consistent with the factory. Refer to `DiskOptionSettings/disk_type` in schema.py.

Reply:
BTW, what's the cause behind? I don't see how it got fixed by this change.

Reply:
I am still now sure, I just see when issue happened, the setting is `arch {'is_allow_set': False, 'items': ['x64']}`